phpstan.neon

```diff
@@ -14,3 +14,4 @@ parameters:
         - '#Parameter \#2 \$dqlPart of method Doctrine\\ORM\\QueryBuilder::add\(\) expects Doctrine\\ORM\\Query\\Expr\\Base, Doctrine\\ORM\\Query\\Expr\\Join\[\] given#' # Fixed in Doctrine's master
         - '#Call to an undefined method Doctrine\\Common\\Persistence\\ObjectManager::getConnection\(\)#'
         - '#Parameter \#1 \$callable of static method Doctrine\\Common\\Annotations\\AnnotationRegistry::registerLoader\(\) expects callable, mixed\[\] given#'
+        - '#Class ApiPlatform\\Core\\Tests\\Bridge\\Doctrine\\Orm\\Filter\\DummyFilter was not found while trying to analyse it - autoloading is probably not configured properly#'
```

Comment:
I like the below test because it's simple, though I feel it's overkill to add a file for my dummy filter which is only used in the test. Is there a way I can avoid adding this? Or maybe it's bad practice and I should move this to a file?